### PR TITLE
Change variable name to avoid conflts with Telegram TL schema

### PR DESCRIPTION
**Pull request by allen0099**
Currently, the api compiler is using `data` as its function argument.  

However, [updateBotCallbackQuery](https://core.telegram.org/constructor/updateBotCallbackQuery) is use the same word `data` as its argument.  
It will cause the problem when the bot gets the callback query with no data (Like the game button).

Trying to fix issue #608 and make game button usable.

### CODE REVIEW

File: pyrogram/types/bots_and_keyboards/callback_query.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ async def _parse(client, callback_query, users) -> "CallbackQuery":
         # ignoring/replacing errors, this way, button clicks will still work.
         try:
             data = callback_query.data.decode()
-        except UnicodeDecodeError:
+        except (UnicodeDecodeError, AttributeError):
```
Comment (delivrance) [inline pyrogram/types/bots_and_keyboards/callback_query.py:113]: `AttributeError`, what for?

**Comment by delivrance** [comment]
You forgot to rename some occurrences (I fixed those). Also, I've renamed to `b` (bytes, buffer) as it used to be.

File: pyrogram/types/bots_and_keyboards/callback_query.py (hunk 1 above)
Comment (allen0099) [reply pyrogram/types/bots_and_keyboards/callback_query.py:113]: Game button will return `data` with `None`

**Comment by Sea-n** [comment]
Hey devs, any update on this PR?
I need this patch too. :bowing_man:

**Comment by asimaranov** [comment]
I faced with this issue to. The following exception occurres if I click game button
IMG_20210812_122033_463. Please fix

**Pull request merged**